src/common/tooltip-utils/constants.ts

```diff
@@ -1,9 +1,15 @@
-import { PointerDirection } from './types'
+import { PointerDirection, TooltipType } from './types'
 
 export const DEFAULT_POINTER_DIRECTION: PointerDirection = 'bottom'
 
+export const TYPE_ROLES: Record<TooltipType, string> = {
+    tooltip: 'tooltip',
+    tourtip: 'region',
+    infotip: ''
```

Comment:
do we need to say ""? it is as good as having null right?

Reply:
This is for typescript not complaining 😢